spec/support/custom_matchers/be_same_time_as.rb

```diff
@@ -1,5 +1,13 @@
 RSpec::Matchers.define :be_same_time_as do |expected|
+  regexp = /(.*_spec\.rb:\d+)/
+  called_from = caller.detect { |line| line =~ regexp }
```

Comment:
This might not be executed at the right time...I think this is executed in order to define the method.  I think you want these two lines *inside* the match block

Reply:
whoops missed that on this one, thanks.